Dockerfile

```diff
@@ -24,7 +24,6 @@ ARG HOST_DIST=$TARGETOS-$TARGETARCH
 RUN adduser -D -u 12345 -g 12345 sm
 
 ADD --chown=sm:sm --chmod=0500 https://github.com/grafana/xk6-sm/releases/download/v0.4.0/sm-k6-${TARGETOS}-${TARGETARCH} /usr/local/bin/sm-k6
```

Comment:
The `xk6-sm` version that ships k6 v0.58.0 is not yet released, right? It might make sense to release that and update the version together with this PR.